mss/mssmain.py

```diff
@@ -423,7 +436,7 @@ def mf_calculator(mass, mass_error=10,
         'massRange': massRange,
         'integerUnsaturation': integerUnsaturation
     }
-
+    time.sleep(1)
```

Comment:
The 1s waiting time might be a large cost of time, maybe there are other packages/apis available?

Reply:
Also a FYI - I used to use local dataset for formula assignment previously, but due to the file size limit it could only cover up to m/z 500 (and not efficient). 

Another option is to try simply pull/request some online calculators?
https://www.sisweb.com/referenc/tools/exactmass.htm